.github/workflows/pythonpublish.yml

```
@@ -17,6 +17,13 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: "3.x"
+
+      - name: Verify version
+        run: |
+          if [[ "$(python3 setup.py --version)" != "${{ github.event.release.tag_name }}" ]]; then
+            exit 1
+          fi
```

Nice little one 👍

---

Should do the same...

```suggestion
      - name: Verify version
        run: |
          [[ "$(python3 setup.py --version)" != "${{ github.event.release.tag_name }}" ]] ```

---

Neat, then this should work as well?
```suggestion
      - name: Verify version
        run: [[ "$(python3 setup.py --version)" != "${{ github.event.release.tag_name }}" ]]
```

---

Yup that works too

---

It did not :(
<https://github.com/NabuCasa/hass-nabucasa/pull/456>